### PR TITLE
[TensorExpr] Kernel: make prim::ConstantChunk handled like other ops.

### PR DESCRIPTION
**Pull request by ZolotukhinM**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #65555
* #65859
* #65852
* #65843
* #65554
* #65553
* #65552
* #65551
* #65550
* __->__ #65549

Previously it had a special handling, with this change it follows the
same mechanism as other ops.

Differential Revision: [D31148924](https://our.internmc.facebook.com/intern/diff/D31148924)

### CODE REVIEW

**Comment by ZolotukhinM** [comment]
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31148924).

File: torch/csrc/jit/tensorexpr/kernel.cpp [hunk 1]
```diff
@@ -2450,29 +2463,21 @@ Tensor TensorExprKernel::computeValue(const torch::jit::Value* v) {
...
-              bufs_.at(n->input(0)), v->offset(), dim, chunks, indices);
-        });
-  }
-
   std::vector<ArgValue> argInputs;
-  if (op != aten::to) {
+  if (op == prim::ConstantChunk) {
+    auto const& n = v->node();
+    argInputs.push_back(toArg(inputs[0]));
+    argInputs.push_back((int64_t)v->offset());
```
Comment (navahgar) [inline torch/csrc/jit/tensorexpr/kernel.cpp:2470]: "static_cast<int64_t>" instead of the traditional cast.

**Review by navahgar — APPROVED** [review]
LGTM

**Comment by ZolotukhinM** [comment]
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31148924).

**Comment by ZolotukhinM** [comment]
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31148924).

**Comment by ZolotukhinM** [comment]
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31148924).

**Comment by ZolotukhinM** [comment]
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31148924).

**Comment by ZolotukhinM** [comment]
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31148924).

**Comment by ZolotukhinM** [comment]
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31148924).

**Pull request closed without merging**